maat-court-data-api/src/main/java/gov/uk/courtdata/exception/GlobalAppLoggingHandler.java

```diff
@@ -68,15 +70,21 @@ public void beforeQueueMessageRec(JoinPoint joinPoint, String message) {
         }
 
         Sentry.configureScope(scope -> {
-            scope.setTag(LoggingData.CASE_URN.getValue(), laaTransactionLogging.getCaseUrn() != null ? laaTransactionLogging.getCaseUrn() : "");
-            scope.setTag(LoggingData.LAA_TRANSACTION_ID.getValue(), laaTransactionIdStr);
-            scope.setTag(LoggingData.MAATID.getValue(), laaTransactionLogging.getMaatId().toString());
+            scope.setTag(LoggingData.CASE_URN.getKey(),
```

Comment:
Do we also need to be clearing out the MDC in the `afterProcessEnds` method for completeness?